ui/apps/platform/src/Containers/Vulnerabilities/VulnerablityReporting/api/useFetchReportHistory.ts

```diff
@@ -0,0 +1,63 @@
+/* eslint-disable no-void */
+import { useCallback, useEffect, useState } from 'react';
+
+import { fetchReportHistory } from 'services/ReportsService';
+import { ReportSnapshot } from 'services/ReportsService.types';
+import { getErrorMessage } from '../errorUtils';
```

Comment:
Would it be possible for us to use the app-wide utility function for error parsing here, too?

https://github.com/stackrox/stackrox/blob/ce65ad1d4e641e48b1660c324896c5a6ba5d7112/ui/apps/platform/src/utils/responseErrorUtils.ts

Reply:
Well, we can actually. I think I don't really need to use async/await here. It's usually cleaner with a promise for a case like this. I think async/await might be better when we need to do multiple calls or something more complicated